docs/source/reward_trainer.mdx

```diff
@@ -0,0 +1,51 @@
+# Reward Modeling
+
+TRL supports custom reward modeling for anyone to perform reward modeling on their dataset and model. 
+
+## Expected dataset format
+
+The reward trainer expects a very specific format for the dataset. The dataset should contain two 4 entries at least
+if you don't use the default `RewardDataCollatorWithPadding` data collator. The entries should be named:
+
+- `input_ids_j`
```

Comment:
Small suggestion: since reward modelling datasets typically come from pairs of chosen/rejected examples, we could use these names to make it clear to users which variables correspond to what:

```suggestion
- `input_ids_chosen`
```

An alternative would be to use good/bad like Anthropic does in their papers

Reply:
Sounds good to me!